### PR TITLE
CMR-7253 Fixed the missed use case where the detailed variable is nex…

### PR DESCRIPTION
**Pull request by eereiter**
…t to other variable facets.

### CODE REVIEW

**Review by ygliuvt — APPROVED** [review]

File: search-app/src/cmr/search/services/query_execution/facets/hierarchical_v2_facets.clj [hunk 1]
```diff
@@ -211,6 +211,16 @@
         links (generate-links-fn value has-siblings? children-values-to-remove)]
     [(v2h/generate-hierarchical-filter-node value count links sub-facets field)]))
 
+(defn- check-if-last-facet-accounted-for?
```
Comment (jaybarra) [inline search-app/src/cmr/search/services/query_execution/facets/hierarchical_v2_facets.clj:214]: Might be able to shorten this to `last-facet-accounted-for?`

**Review by jaybarra — APPROVED** [review]
